release/README.md

```diff
@@ -1,33 +1,36 @@
-# How to make a fortio release
+## How to make a fortio release
 
-- Make sure `periodic/periodic.go`'s `Version` is newer than https://github.com/istio/fortio/releases
+- Make sure `version/version.go`'s `major`/`minor`/`patch` is newer than https://github.com/istio/fortio/releases
 
 - Make a release there and document the changes since the previous release
 
-- Make sure to use the same tag format (e.g "0.4.3" - note that there are no "v" in the tag to be consistent with the rest istio)
+- Make sure to use the same tag format (e.g "v0.7.1" - note that there is "v" in the tag unlike the rest of istio)
```

Comment:
What is the reason for adding "v" to the tag?

Reply:
it’s not new just updating the doc

the interesting question would be why istio doesn’t have a v - it’s pretty standard semver to have tags vX.Y.Z

Reply:
I don't see any mention of a v prefix in the semver doc, and don't know of anything else that uses it either (it's a bit redundant to say "version v0.7.1").  I don't see it used in docker tags (including for fortio).

Anyway, I haven't been clear about who/what is supposed to update the fortio tags in the istio repo (e.g., updateVersions.sh).    Most cases I see lazily use "istio/fortio:latest" (or equivalent), which is fine while on master but it's got potential to cause problems when someone is working with an older branch or release of istio.  Occasionally I come across files like samples/httpbin/sample-client/fortio-deploy.yaml that hard-code a specific version and I'm not sure if it should just be using "latest" like everyone else.

Reply:
The version in the code is 0.7.0, likewise in the tgz and the docker tag but the github tag is "v0.7.0" maybe I need to clarify which tag is being talked about here (ie git tag)

Example of other projects using vX.Y.Z for their tags : 
https://github.com/grpc/grpc-go/releases
https://github.com/kubernetes/kubernetes/releases

Re updating tag in istio/istio - fortio ends up in vendor now, but that tag in istio.VERSION is used by some e2e test to install an image; :latest would work as long as I keep master pristine